src/Compilers/CSharp/Portable/Syntax/ArgumentSyntax.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Syntax
+{
+    public sealed partial class ArgumentSyntax
+    {
+        public SyntaxToken RefOrOutKeyword
```

Comment:
:bulb: It would be helpful to add documentation comments to the property and method here explicitly stating that they simply call the other form.